### PR TITLE
Draft of cover letter

### LINKED ISSUE #174: Cover Letter
We need a [cover letter](https://www.nature.com/srep/publish/guidelines#cover-letter).

@tylerjereddy You've mentioned that you know our editorial contact, Syma Khalid. Did you plan on writing the cover letter or should I draft it? I was hoping we could submit by Wednesday.

### PR DESCRIPTION
**Pull request by tylerjereddy**
Fixes #174 

Instructions: https://www.nature.com/srep/publish/guidelines#cover-letter

Everyone has different views on how to write these things. I probably didn't do enough to get the letter PDF to show up in CI but local `make covlett` should suffice.

TODO:

- [ ] fix up affiliations / contact info & details (some of that may just happen right before submission outside the repo if someone prefers to keep some of that more private)
- [x] I don't think we have specific reviewer exclusions? Does anyone really really hate SciPy?
- [x] what are good suggestions for peer reviewers--maybe someone on the Julia team or?

### CODE REVIEW

**Comment by rgommers** [comment]
> I don't think we have specific reviewer exclusions? Does anyone really really hate SciPy?

Nah, everyone loves us:)

> what are good suggestions for peer reviewers--maybe someone on the Julia team or?

that works, or also others in the Python community that are a bit further removed from SciPy (e.g. Sage, or some well-known academics who are users of SciPy or use it for education)

**Comment by rgommers** [comment]
Reads well to me, thanks Tyler

File: scipy-1.0/covlett.tex [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We are submitting the enclosed manuscript describing a milestone
+1.0 release of SciPy for consideration to be published in
+\emph{Scientific Reports}. SciPy is the fundamental scientific 
+algorithm computing library in the Python programming language. 
+Hundreds of experts have contributed to our library, which
+has been download millions of times and has over 100,000 dependent
```
Comment (jnothman) [inline scipy-1.0/covlett.tex]:
```suggestion
has been downloaded millions of times and has over 100,000 dependent
```

File: scipy-1.0/covlett.tex [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+SO17 1BJ, U.K.}
+
+\greetto{Dear Prof. Khalid,}
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We are submitting the enclosed manuscript describing a milestone
+1.0 release of SciPy for consideration to be published in
+\emph{Scientific Reports}. SciPy is the fundamental scientific
```
Comment (jnothman) [inline scipy-1.0/covlett.tex]: I find "scientific algorithm computing library" a bit hard to parse. is it:
[ ] scientific (algorithm (computing library))
[ ] scientific ((algorithm computing) library)
[ ] (scientific (algorithm computing)) library
[x] (scientific algorithm) (computing library)
[ ] ((scientific algorithm) computing) library

File: scipy-1.0/covlett.tex [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+\greetto{Dear Prof. Khalid,}
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We are submitting the enclosed manuscript describing a milestone
+1.0 release of SciPy for consideration to be published in
+\emph{Scientific Reports}. SciPy is the fundamental scientific 
+algorithm computing library in the Python programming language.
```
Comment (apbard) [inline scipy-1.0/covlett.tex]: I agree with @jnothman...maybe just remove `algorithm`
```suggestion
computing library in the Python programming language. 
```

File: scipy-1.0/covlett.tex [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+We are submitting the enclosed manuscript describing a milestone
+1.0 release of SciPy for consideration to be published in
+\emph{Scientific Reports}. SciPy is the fundamental scientific 
+algorithm computing library in the Python programming language. 
+Hundreds of experts have contributed to our library, which
+has been download millions of times and has over 100,000 dependent
+code repositories.
+
+We have specifically chosen \emph{Scientific Reports} because
+the breadth of acceptable topics matches the breadth of algorithms
```
Comment (apbard) [inline scipy-1.0/covlett.tex]: just to avoid repetition
```suggestion
the breadth of acceptable topics matches the variety of algorithms
```

File: scipy-1.0/covlett.tex (hunk 2 above)
Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: fixed by applying the suggestion below, thanks

**Comment by mdhaber** [comment]
The author of Sage Math is William Stein. His email address can be found [here](http://www.sagemath.org/contact.html).

**Comment by mdhaber** [comment]
I was looking for a SciPy equivalent for Julia. I couldn't find something with the breadth of SciPy, but [Optim.jl](https://github.com/JuliaNLSolvers/Optim.jl) is one of the two optimization packages mentioned on the [Julia website](https://julialang.org/). The top contributor is [Jihao Chen](https://github.com/JuliaMath/IterativeSolvers.jl/graphs/contributors), who has a [solid academic track record](https://scholar.google.com/citations?user=TQYNuFAAAAAJ&hl=en&oi=ao). His email address is listed [here](https://jiahao.github.io/#contact).

**Comment by mdhaber** [comment]
[Søren Hauberg](https://scholar.google.com/citations?user=M1fMGOMAAAAJ&hl=en&oi=sra) was a coauthor of the [Octave Manual](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=GNU+octave&btnG=). His email address is listed [here](http://www2.compute.dtu.dk/~sohau/contact/).

**Comment by mdhaber** [comment]
[Kazunori Akiyama](https://scholar.google.com/citations?hl=en&user=e3DC1FwAAAAJ) is the first named author of the [event horizon imaging paper ](https://iopscience.iop.org/article/10.3847/2041-8213/ab0c57/meta) that cites SciPy. His email address is listed [here](https://bhi.fas.harvard.edu/people/kazunori-akiyama).

Anyway, those are just some thoughts. Don't know much about any of them; just brainstorming.

**Comment by tylerjereddy** [comment]
Ok, I drafted in 3 suggested reviewers. Feel free to change them though. I didn't check if the letter still compiles since I'm on a different machine at the moment; should be ok though.

Don't let me hold you back if the plan is to submit today / in a few hours.

**Comment by mdhaber** [comment]
Probably won't submit until the weekend. There are a few open PRs that need to be reviewed or closed (and #178 needs to be finished), and I need to spend the day on the CZI proposal.

But is this ready to go from your perspective? Title is still WIP, but it looks good to me.

**Comment by tylerjereddy** [comment]
Should be close -- if we add someones email + phone number or something that might happen offline just before submitting

**Comment by mdhaber** [comment]
Sorry, I didn't understand the bit about email and phone number?
Also, can I merge this PR?

**Comment by rgommers** [comment]
> Kazunori Akiyama is the first named author of the event horizon imaging paper that cites SciPy. His email address is listed here.

may be more appropriate than an Octave person. Octave was important back when Matlab ruled the world, but probably not too relevant. having one listed review from the Python space would make sense.

LGTM otherwise. I'll send both of you my phone number

**Comment by tylerjereddy** [comment]
@mdhaber Yeah, I just meant email / phone to add relatively complete contact info to the final cover letter before submitting. 

I'll try to address Ralf's comment a bit later -- just finished the bulk of unpacking after relocating back to NM.. bad timing!

**Comment by tylerjereddy** [comment]
Ok, I addressed Ralf's comment. We'll need a little whitespace / formatting clean-up to confine to a single page, but otherwise looking ok I think.

**Comment by tylerjereddy** [comment]
I pushed in some vertical spacing improvements so that we don't overflow to a second page; also added a link to the spacing adjustment instructions just in case.

File: scipy-1.0/covlett.tex [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+computing library in the Python programming language. 
+Hundreds of experts have contributed to our library, which
+has been downloaded millions of times and has over 100,000 dependent
+code repositories.
+
+We have specifically chosen \emph{Scientific Reports} because
+the breadth of acceptable topics matches the variety of algorithms
+our library provides. High-profile manuscripts citing SciPy range
+from the biosciences to recent reports on gravitational waves and
+black holes. These citations target a website, and we now think
```
Comment (stefanv) [inline scipy-1.0/covlett.tex]: "These citation target a website, and we now think it is well past time..."  This latter parts reads a bit strangely to me, but my brain doesn't have an alternative to offer right now.

Comment (ilayn) [reply scipy-1.0/covlett.tex]: Something like this? 

> Since these citations, for lack of a better alternative, cite the website that SciPy currently located, we would like to provide an academic article for this milestone which also signals better the SciPy's maturity level.

I ran out of English towards the end though.

Comment (stefanv) [reply scipy-1.0/covlett.tex]: Ah, that gives me a good start!

> Lacking a better alternative, these citations all point to the SciPy website; instead, we think the package now (would be better served | merits | deserves) a formal publication in a highly visible journal.

Comment (tylerjereddy) [reply scipy-1.0/covlett.tex]: Ok, I've applied Stefan's suggested revision of this sentence

**Comment by tylerjereddy** [comment]
With the latest revision I think we're at 4 approving core devs, and Matt & I will try to submit today, so in this goes.
